### PR TITLE
Charlotte/new pick up

### PR DESCRIPTION
**Pull request by CharlotteLaw**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### Description
temp PR (not fully finished) 

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"

## How to review
[//]: # "Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # "Optional - copy links to any tutorials or documentation that was useful to you when working on this PR"

### Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

[//]: # "This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation."

### CODE REVIEW

File: src/app/orderConfirmationPickUp/styles.ts [hunk 1]
```diff
@@ -22,36 +23,24 @@ export const OutterFavoriteDiv = styled.div`
   border-radius: 10px;
   background: var(--White, #fff);
   box-shadow: 0px 1px 4px 1px rgba(0, 0, 0, 0.2);
-  width: 800px;
+<<<<<<< HEAD
```
Comment (BuyankhuuTsCAl) [inline src/app/orderConfirmationPickUp/styles.ts]: is this a merge conflict?

File: src/app/orderConfirmationPickUp/styles.ts [hunk 2]
```diff
@@ -110,18 +99,17 @@ export const Label = styled.p`
   margin-top: 20px;
 `;
 
-export const LabelBox1 = styled.div`
-  width: 250px;
-  height: 100%;
-  padding: 20px;
+export const ImageDiv = styled.p`
```
Comment (BuyankhuuTsCAl) [inline src/app/orderConfirmationPickUp/styles.ts]: why is this a p tag

File: src/app/orderConfirmationPickUp/styles.ts [hunk 3]
```diff
@@ -254,31 +242,41 @@ export const WhiteBackgroundDiv = styled.div`
   box-shadow: 0px 4px 7px 0px rgba(0, 0, 0, 0.1);
 `;
 
+export const Wrapper2 = styled.div`
+  display: flex;
+  flex-direction: col;
```
Comment (BuyankhuuTsCAl) [inline src/app/orderConfirmationPickUp/styles.ts:263]: does col work?

File: src/app/orderConfirmationPickUp/styles.ts [hunk 4]
```diff
@@ -254,31 +242,41 @@ export const WhiteBackgroundDiv = styled.div`
...
+  justify-content: space-around;
+`;
+
 export const BottomColumnDiv = styled.div`
   display: flex;
   flex-direction: row;
   align-items: space-evenly;
   justify-content: space-around;
   width: 100%;
+<<<<<<< HEAD
```
Comment (BuyankhuuTsCAl) [inline src/app/orderConfirmationPickUp/styles.ts]: merge conflict

File: src/app/orderConfirmationPickUp/styles.ts [hunk 5]
```diff
@@ -287,22 +285,29 @@ export const PageDiv = styled.div`
...
-  width: 467px;
+  width: 475px;
   height: auto;
   max-height: 100%;
   padding: 36px 34px;
   gap: 33px;
   max-height: 100%;
-  margin-top: 205px;
+<<<<<<< HEAD
+  margin-top: 75px;
```
Comment (BuyankhuuTsCAl) [inline src/app/orderConfirmationPickUp/styles.ts:319]: conflict

File: src/app/orderPage/page.tsx [hunk 6]
```diff
@@ -77,39 +97,82 @@ export default function OrderPage() {
...
+                          >
+                            Quantity:
+                          </span>
+                          <span
+                            style={{
+                              display: 'inline-block',
+                              fontSize: '16px',
+                            }}
+                          >
+                            {product.quantity}
```
Comment (BuyankhuuTsCAl) [inline src/app/orderPage/page.tsx]: is there styling on this text?

File: src/app/orderPage/page.tsx [hunk 7]
```diff
@@ -77,39 +97,82 @@ export default function OrderPage() {
...
+                      <LabelBox>
+                        <div>
+                          <span
+                            style={{
+                              display: 'inline-block',
+                              fontWeight: 'bold',
+                              marginRight: '4px',
+                            }}
+                          >
+                            Quantity:
```
Comment (BuyankhuuTsCAl) [inline src/app/orderPage/page.tsx]: same question

File: src/app/profileScreen/styles.ts [hunk 8]
```diff
@@ -112,8 +130,16 @@ export const HeartIcon = styled(Heart)`
 `;
 
 export const BackButtonDiv = styled.div`
+<<<<<<< HEAD
```
Comment (BuyankhuuTsCAl) [inline src/app/profileScreen/styles.ts]: merge conflict

File: src/app/orderConfirmationPickUp/styles.ts (hunk 3 above)
Comment (CharlotteLaw) [reply src/app/orderConfirmationPickUp/styles.ts:263]: yes
